docs/device_api.md

```diff
@@ -333,6 +333,21 @@ device" section).
 
 If you take a look to the previous device example, you can find that a "ping" command was defined.
 Any update on this attribute “Ping” at the NGSI entity in the ContextBroker will send a command to your device.
+For instance, to send the "Ping" command with value "Ping request" you could use the following operation in the
+ContextBroker API:
+
+```
+PUT /v2/entities/[ENTITY_ID]/attrs/ping
+
+{
+  "value": "Ping request",
+  "type": "command"
+}
+
+```
+
```

Comment:
I would add something like, "Please note that the the verb should be PUT or PATCH (not POST) in v2 and the action should be UPDATE (not APPEND) in v1 "

Reply:
Fixed in a3d5550